perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -1023,56 +1104,70 @@ protected boolean hasCandidateExistingMember(Candidate candidate, RichMember ric
 	 * This method run in separate transaction.
 	 */
 	public List<String> synchronizeGroup(PerunSession sess, Group group) throws InternalErrorException, MemberAlreadyRemovedException, AttributeNotExistsException, WrongAttributeAssignmentException, ExtSourceNotExistsException, WrongAttributeValueException, WrongReferenceAttributeValueException, GroupOperationsException, GroupNotExistsException {
-		//needed variables for whole method
+		// Needed variables for whole method
 		List<String> skippedMembers = new ArrayList<>();
 		ExtSource source = null;
 		ExtSource membersSource = null;
 
 		try {
 			log.info("Group synchronization {}: started.", group);
 
-			//Initialization of group extSource
+			// Initialization of group extSource
 			source = getGroupExtSourceForSynchronization(sess, group);
 
-			//Initialization of groupMembers extSource (if it is set), in other case set membersSource = source
+			// Initialization of groupMembers extSource (if it is set), in other case set membersSource = source
 			membersSource = getGroupMembersExtSourceForSynchronization(sess, group, source);
 
-			//Prepare info about userAttributes which need to be overwrite (not just updated) and memberAttributes which need to be merged )not overwrite
+			// Prepare info about userAttributes which need to be overwrite (not just updated) and memberAttributes which need to be merged )not overwrite
 			List<String> overwriteUserAttributesList = getOverwriteUserAttributesListFromExtSource(membersSource);
 			List<String> mergeMemberAttributesList = getMemberAttributesListToBeMergedFromExtSource(membersSource);
 
-			//Get info about type of synchronization (with or without update)
+			// Get info about type of synchronization (with or without update)
 			boolean lightweightSynchronization = isThisLightweightSynchronization(sess, group);
 
 			log.info("Group synchronization {}: using configuration extSource for membership {}, extSource for members {}", new Object[] {group, membersSource, membersSource.getName()});
 
-			//Prepare containers for work with group members
+			// Prepare containers for work with group members
 			List<Candidate> candidatesToAdd = new ArrayList<>();
 			Map<Candidate, RichMember> membersToUpdate = new HashMap<>();
 			List<RichMember> membersToRemove = new ArrayList<>();
-
-			//get all actual members of group
+			// Get all actual members of group
 			List<RichMember> actualGroupMembers = getPerunBl().getGroupsManagerBl().getGroupRichMembers(sess, group);
+			// Subjects gained from external source
+			List<Map<String, String>> subjects = new ArrayList<>();
 
+			String status;
 			if(lightweightSynchronization) {
-				categorizeMembersForLightweightSynchronization(sess, group, source, membersSource, actualGroupMembers, candidatesToAdd, membersToRemove, skippedMembers);
+				status = GroupsManager.GROUP_SYNC_STATUS_LIGHTWEIGHT;
```

Comment:
Because I use it as a parameter of getSubjectsFromExtSource method. Then I change value of "status" attribute with returned value of mentioned method.